src/test/resources/database/categories/add-three-default-categories.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM categories;
```

Comment:
Let's move it to the different sql script, it's not obvious that script with name "add...." also delete something